ui/.eslintrc.yml

```diff
@@ -39,3 +39,4 @@ rules:
     - singleline:
         delimiter: 'semi'
         requireLast: false
+  '@typescript-eslint/no-dynamic-delete': 0
```

Comment:
Not fixing these? Maybe only disable where it makes sense instead of globally?

https://github.com/typescript-eslint/typescript-eslint/blob/HEAD/packages/eslint-plugin/docs/rules/no-dynamic-delete.md